From 4bbc10614f3431c37e6352f5a6ea5c693c31021e Mon Sep 17 00:00:00 2001
From: Maria Matejka <mq@ucw.cz>
Date: Tue, 4 Feb 2020 10:11:16 +0100
Subject: [PATCH] Added missing extern
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Thanks to Robert Scheck <bird@robert-scheck.de> who reported it
and Toke Høiland-Jørgensen <toke@toke.dk> who suggested this patch.

[Retrieved from:
https://gitlab.nic.cz/labs/bird/-/commit/4bbc10614f3431c37e6352f5a6ea5c693c31021e]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 nest/route.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nest/route.h b/nest/route.h
index d2a07f09..b927db5f 100644
--- a/nest/route.h
+++ b/nest/route.h
@@ -458,7 +458,7 @@ typedef struct rta {
 					   protocol-specific metric is availabe */
 
 
-const char * rta_dest_names[RTD_MAX];
+extern const char * rta_dest_names[RTD_MAX];
 
 static inline const char *rta_dest_name(uint n)
 { return (n < RTD_MAX) ? rta_dest_names[n] : "???"; }
-- 
2.26.2

